### PR TITLE
Add Kotlin code for "Communication Android" documentation 

### PR DESCRIPTION
**Pull request by junaidn**
This PR includes:
Sample code of Kotlin for `docs/communication-android` doc.

### CODE REVIEW

**Comment by junaidn** [comment]
@cortinico @ShikaSD @mdvacca

File: docs/communication-android.md [hunk 1]
```diff
@@ -39,6 +45,29 @@ public class MainActivity extends ReactActivity {
...
+
+```kotlin
+class MainActivity : ReactActivity() {
+    override fun createReactActivityDelegate(): ReactActivityDelegate {
+        return object : ReactActivityDelegate(this, mainComponentName) {
+            override fun getLaunchOptions(): Bundle {
+                val initialProperties = Bundle()
+                val imageList =
+                    ArrayList(listOf("http://foo.com/bar1.png", "http://foo.com/bar2.png"))
+                initialProperties.putStringArrayList("images", imageList)
```
Comment (cortinico) [inline docs/communication-android.md:60]: I'd reword to:

```kotlin
val initialProperties = Bundle().apply {
    putStringArrayList("images", imageList)
}
```

File: docs/communication-android.md [hunk 2]
```diff
@@ -39,6 +45,29 @@ public class MainActivity extends ReactActivity {
...
+<TabItem value="kotlin">
+
+```kotlin
+class MainActivity : ReactActivity() {
+    override fun createReactActivityDelegate(): ReactActivityDelegate {
+        return object : ReactActivityDelegate(this, mainComponentName) {
+            override fun getLaunchOptions(): Bundle {
+                val initialProperties = Bundle()
+                val imageList =
+                    ArrayList(listOf("http://foo.com/bar1.png", "http://foo.com/bar2.png"))
```
[the remark this answers is not in the document]
Comment (junaidn) [reply docs/communication-android.md]: Done

File: docs/communication-android.md (hunk 1 above)
Comment (junaidn) [reply docs/communication-android.md]: Done

File: docs/communication-android.md [hunk 3]
```diff
@@ -66,6 +99,20 @@ updatedProps.putStringArrayList("images", imageList);
...
 ```
 
+</TabItem>
+
+<TabItem value="kotlin">
+
+```kotlin
+var updatedProps: Bundle = mReactRootView.getAppProperties()
+var imageList: ArrayList<String> =
+    ArrayList<String>(listOf("http://foo.com/bar3.png", "http://foo.com/bar4.png"))
```
[the remark this answers is not in the document]
Comment (junaidn) [reply docs/communication-android.md]: Done

File: docs/communication-android.md [hunk 4]
```diff
@@ -66,6 +97,19 @@ updatedProps.putStringArrayList("images", imageList);
 mReactRootView.setAppProperties(updatedProps);
 ```
 
+</TabItem>
+
+<TabItem value="kotlin">
+
+```kotlin
+var updatedProps: Bundle = mReactRootView.getAppProperties()
```
Comment (cortinico) [inline docs/communication-android.md:105]:
```suggestion
var updatedProps: Bundle = reactRootView.getAppProperties()
```

**Review by cortinico — APPROVED** [review]
Left a small nit. Other than that, LGTM 👍

**Comment by cortinico** [comment]
Thansk for doing this @junaidn 🙏
